### ISSUE TITLE
Mobile: Some items missing accessibility labels

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#8796]
See https://github.com/laurent22/joplin/issues/8794

In particular, the "clear" button to the right of the search button, and the actions menu button are missing labels.
